utils/hooks/useSendQuoteRequest.ts

```diff
@@ -13,19 +13,17 @@ interface FormData {
   details?: string;
 }
 
-interface UseSendQuoteRequestReturn {
-  register: ReturnType<typeof useForm>["register"];
-  handleSubmit: ReturnType<typeof useForm>["handleSubmit"];
+type UseSendQuoteRequestReturn = {
+  register: ReturnType<typeof useForm<FormData>>["register"];
+  handleSubmit: ReturnType<typeof useForm<FormData>>["handleSubmit"];
   formState: {
-    errors: {
-      [key in keyof FormData]?: string;
-    };
+    errors: FieldErrors<FormData>;
```

Comment:
Typescript was unhappy because the errors object on formState has a different shape from what you were trying to define here. Per ChatGPT, 

> The error message suggests that the type mismatch occurs because TypeScript expects errors to have properties name, email, industry, and details, each of which should be a string or undefined.
> 
> To resolve this, let's ensure that the structure of the errors object matches TypeScript's expectations. We can do this by using the FieldErrors type provided by react-hook-form for the errors object.

Reply:
lol chatGPT suggested 
```
    errors: {
      [key in keyof FormData]?: string;
    };
```
to me to fix the original TS errors